src/Components/Server/src/Builder/ComponentEndpointConventionBuilder.cs

```diff
@@ -10,11 +10,11 @@ namespace Microsoft.AspNetCore.Builder
     /// </summary>
     public sealed class ComponentEndpointConventionBuilder : IHubEndpointConventionBuilder
     {
-        private readonly IEndpointConventionBuilder _endpointConventionBuilder;
+        private readonly IEndpointConventionBuilder [] _endpointConventionBuilders;
 
-        internal ComponentEndpointConventionBuilder(IEndpointConventionBuilder endpointConventionBuilder)
+        internal ComponentEndpointConventionBuilder(params IEndpointConventionBuilder [] endpointConventionBuilder)
```

Comment:
any real reason for params here? You always pass two.

Reply:
I was just following the pattern signalr follows. It feels pretty straightforward and its all internal. Do you feel strongly against it? I've changed it, but I don't think it's important at all